conf/messages.en

```diff
@@ -248,9 +248,11 @@ cf.authorities.next.msg = Your list will be available from the ''Find accounts y
 cf.contact-details.title=Your contact details
 cf.contact-details.company-details=Company details
 cf.contact-details.name=Name
-cf.contact-details.address=Address
+cf.contact-details.address=Contact address
 cf.contact-details.email-address=Email address
-cf.contact-details.report-change-link=Report a change to your company details (opens in new tab)
+cf.contact-details.report-change-link=fill in an enquiry form (opens in a new tab)
+cf.contact-details.report-change-pre-link-text=This is the contact address you gave us when you registered for your EORI number. You can
```

Comment:
Could you please remove 'You can' from this sentence as it should be part of newTabLink pre message?

'This is the contact address you gave us when you registered for your EORI number.' should be in a separate paragraph followed by newTabLink

Reply:
what I meant is, that the sentence 'This is the contact address you gave us when you registered for your EORI number.' should be a separate message and 'You can' should not be joined with mentioned sentence as these are two different sentences and their Welsh translation would be different. Yes it will not be a separate paragraph (sorry got the term wrong earlier) so it can stay with only a message key.

Right now your Welsh translation is incorrect. Check in UI